java-repo-tools/scripts/test-localhost.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+#     test-localhost.sh deployment-type path/to/project -- [maven arguments]
+#
+# This script runs a localhost server Maven plugin and verifies that a request
+# to http://localhost:8080/ does not return an error code.
+
+print_usage () {
+  echo "Usage:" >&2
+  echo "  $0 server-type path/to/project [-- maven arguments]" >&2
+  echo >&2
+  echo "server-type can be any of the following:" >&2
+  echo "  appengine" >&2
```

Comment:
Standard. The names match those of the Maven plugins, but I suppose that could as we add more kinds of samples.